overlays/bases/deployments/kustomization.yaml

```diff
@@ -52,3 +52,7 @@ resources:
   - base/precise-code-intel/bundle-manager.Service.yaml
   - base/precise-code-intel/worker.Deployment.yaml
   - base/precise-code-intel/worker.Service.yaml
+  - base/cadvisor/cadvisor.DaemonSet.yaml
```

Comment:
i think we need to move all of them to rbac-roles, so non-privileged can avoid this whole cadvisor

Reply:
looks like this means we don't have to do any trimming out of cadvisor? 15fa9cc